exp/query/config.go

```diff
@@ -13,7 +13,7 @@ package query
 
 // Config represents any type of configs and flags to control the querier
 type Config struct {
-	GRPCListenAddr string `long:"grpc.listen" description:"gRPC address that query node listens at" default:"0.0.0.0:9091"`
```

Comment:
9091 is taken on prometheus server on our helm chart. so remapping to 7071 as sane default.